### DISCUSSION TITLE
Remove the http-equiv meta tag: x-ua-compatible

### DISCUSSION THREAD

**Discussion by cristianCeamatuAssist** [gatsbyjs/gatsby#30480] [Help]
A SEO report from https://www.dareboost.com/ is asking to remove the <meta http-equiv="x-ua-compatible" content="IE=edge"> as stated below:
```
HTTP headers are more efficient than the http-equiv meta tags.

The <meta http-equiv=""/> tags
The http-equiv meta tags provide the web browser with information similar to HTTP headers. For example, defining the meta <meta http-equiv="content-type"/> is the same as sending the HTTP Content-Type header.

There are two disadvantages in using http-equiv meta tags:

Going through the meta requires to interpret the beginning of the HTML page, which is slower than going through the HTTP headers in terms of performance
If the HTTP header is already present, the meta is ignored
In which cases are the <meta http-equiv=""/> useful?
Only one case can justify the presence of these meta tags: if you don’t have access to the configuration of your server, and that is to say to the HTTP headers.

However, we recommend that you use a configurable server so that you can establish the most efficient site possible.


This page contains 1 http-equiv meta tag. If possible, you should replace it:

x-ua-compatible
``` 
I could not find a solution until now to configure Gatsby not to add that meta tag, is there any way? I am currently removing it from the `public` folder before each deployment.

**Comment by LekoArts** [comment]
I'd encourage you do research on the internet if this advice actually holds truth (e.g. regarding performance). If you absolutely must, you can edit it here: https://www.gatsbyjs.com/docs/custom-html/

**Reply by cristianCeamatuAssist** [reply]
That is amazing! Thank you @LekoArts, the client wants these requests fixed so we just apply them to make him happy 😄

**Reply by cristianCeamatuAssist** [reply]
Do you happen to know if I can change the way gatsby bundles pages found in `src/pages` named with `.html.js`, for example, I need Gatsby to bundle `src/pages/contact-us.html.js` to `build/contact-us.html` because now it bundles it to `build/contact-us.html/index.html` . I already asked the question [here](https://github.com/gatsbyjs/gatsby/discussions/30459) and explained the reason I need this.
